charts/pgadmin4/values.yaml

```diff
@@ -25,6 +25,8 @@ priorityClassName: ""
 
 service:
   type: ClusterIP
+  clusterIP: ""
```

Comment:
Can we add these commented out?

Reply:
Theoretically - yes, but it should be checked what the default value is there.